yt_dlp/extractor/wrestleuniverse.py

```diff
@@ -0,0 +1,231 @@
+import base64
+import binascii
+import json
+import time
+
+from .common import InfoExtractor
+from ..utils import (
+    ExtractorError,
+    int_or_none,
+    jwt_decode_hs256,
+    traverse_obj,
+    url_or_none,
+)
+
+
+class WrestleUniverseBaseIE(InfoExtractor):
+    _VALID_URL_TMPL = r'https?://(?:www\.)?wrestle-universe\.com/(?:(?P<lang>\w{2})/)?%s/(?P<id>\w+)'
+    _API_PATH = None
+    _TOKEN = None
+    _TOKEN_EXPIRY = None
+
+    def _get_token_cookie(self):
```

Comment:
Isn't it simpler to put this in `_real_intitialize`?

Reply:
See below.